be/src/vec/exec/scan/vscan_node.h

```diff
@@ -34,7 +34,9 @@ class VSlotRef;
 class VScanNode : public ExecNode {
 public:
     VScanNode(ObjectPool* pool, const TPlanNode& tnode, const DescriptorTbl& descs)
-            : ExecNode(pool, tnode, descs), _runtime_filter_descs(tnode.runtime_filters) {}
+            : ExecNode(pool, tnode, descs), _runtime_filter_descs(tnode.runtime_filters) {
+        _runtime_filter_descs = tnode.runtime_filters;
```

Comment:
Duplicated?

Reply:
Double checked, it is duplicated. This line ```_runtime_filter_descs(tnode.runtime_filters)``` was added in your new pr, will remove the duplicated line.